packages/playwright-core/src/server/injected/injectedScript.ts

```diff
@@ -1038,6 +1038,11 @@ export class InjectedScript {
         if (element.nodeName !== 'INPUT' && element.nodeName !== 'TEXTAREA' && element.nodeName !== 'SELECT')
           throw this.createStacklessError('Not an input element');
         received = (element as any).value;
+      }  else if (expression === 'to.contain.class') {
+        return {
```

Comment:
We should either support matching multiple class names or throw if the given one contains a whitespace character?

Reply:
So, I made what you proposed: throw on any whitespace.

Reply:
I had a discussion with the team about it, so basically the semantics of it should be the same as in toHaveClass

```
expect(locator).toContainClass("foobar") -> makes sure locator has foobar
expect(locator).toContainClass("foobar hello") -> makes sure locator has foobar and hello class
expect(locator).toContainClass(["foobar hello", "foobar"]) -> makes sure that the first element has class foobar and hello and the second only foobar.
```

See the docs for toHaveClass: https://playwright.dev/docs/next/test-assertions#locator-assertions-to-have-class

Reply:
done